### ISSUE TITLE
Server fetch fails for search

### ISSUE THREAD

**Issue by danivijay** [danivijay/tab-tracker#4]
```js
if (search) {
  songs = await Song.findAll({
    where: {
      $or: [
        'title', 'artist', 'genre', 'album'
      ].map(key => ({
        [key]: {
          $like: `%${search}%`
        }
      }))
    }
  })
}
```

**Comment by codyseibert** [comment]
"fails for search".  Could you explain what is failing?  Is there an error printed out?  Does it return an empty array?

**Comment by codyseibert** [comment]
here is the issue.  You are probably using a newer version of sequelize which does queries a bit different.  Instead of using $like or $or, you need to use the Sequelize.Op object with Op.or, Op.like.

```
// include these at the top now
const Sequelize = require('sequelize');
const Op = Sequelize.Op;

        songs = await Song.findAll({
          where: {
            [Op.or]: [
              'title', 'artist', 'genre', 'album'
            ].map(key => ({
              [key]: {
                [Op.like]: `%${search}%`
              }
            }))
          }
        })

```

**Comment by danivijay** [comment]
@codyseibert solved the issue by downgrading sequelize to 4.8.0. I know this is not a good solution, but really want to finish your tutorial without errors ASAP. 

**Super-excited to see your quick response! Thank you very much!!**

**Comment by danivijay** [comment]
@codyseibert thanks for the solution. For now, going with older version. Will update once the project completed. 

My first priority is to build tab-manager ASAP. Deep dive into sequelize is scheduled for later!

**Your tutorial is my first stepping stone into the world of JavaScript. Expecting more with VueJS, ExpressJS and other new web techs in future. Thank you!**
